### PR TITLE
ONE: Normalize markdown

### PR DESCRIPTION
**Pull request by kagarmoe**
My scripts don't work correctly on poorly-formed markdown. This PR standardizes H2s, demotes some headers, standardizes title case for headers, and some general linting.

### CODE REVIEW

File: www/source/docs/using-habitat.html.md.erb [hunk 1]
```diff
@@ -111,23 +111,23 @@ Running the `hab svc load` subcommand multiple times with different package iden
...
+### Stopping a Running Service
 
 Sometimes you need to stop a running service for a period of time, for example during a maintenance outage. Rather than completely removing a service from supervision, you can use the `hab svc stop` subcommand which will shut down the running service and leave it in this state until you start it again with the `hab svc start` subcommand, explained next. This means that all service-related options such as service topology, update strategy, etc. are preserved until the service is started again. For example, to stop the running `core/redis` service:
 
 ```bash
 $ hab svc stop core/redis
 ```
 
-## Restarting a stopped service
+### Restarting a Stopped Aervice
```
Comment (mwrock) [inline www/source/docs/using-habitat.html.md.erb]: Service

File: www/source/docs/best-practices.html.md.erb [hunk 2]
```diff
@@ -40,7 +40,7 @@ sudo useradd -g hab hab
 
 Finally, you will need to wire Chef Habitat up to your systems init system. This may be SysVinit, SystemD, runit, etc. The details will be different for each system, but in the end, you must call `hab sup run`.
 
-#### Running under SystemD
+#### Running Under SystemD
```
Comment (mwrock) [inline www/source/docs/best-practices.html.md.erb]: I don't think we need to capitalize prepositionis

File: www/source/docs/using-habitat.html.md.erb (hunk 1 above)
Comment (kagarmoe) [inline www/source/docs/using-habitat.html.md.erb]:
```suggestion
### Restarting a Stopped Service
```

File: www/source/docs/best-practices.html.md.erb (hunk 2 above)
Comment (kagarmoe) [inline www/source/docs/best-practices.html.md.erb]:
```suggestion
#### Running under SystemD
```

**Pull request merged**